tasks/main.yml

```diff
@@ -16,9 +16,17 @@
        | selectattr('state', 'equalto', 'absent') | list) != libvirt_vms
 
 # Libvirt requires qemu-img to create qcow2 files.
-- name: Ensure qemu-img is installed
+- name: Ensure qemu-img and ovmf is installed
+  vars:
+    is_efi: >-
+      (libvirt_vms | selectattr('state', 'defined')
+        | selectattr('state', 'equalto', 'absent') |
```

Comment:
Should this be rejectattr?

Reply:
I just copy pasted the from L14, but you're right - we don't need to install ovmf if we only want to remove VMs (I would assume it's already installed then).

Reply:
It's a bit tricky also because the default for `state` is `present`, but the first selectattr only looks for VMs with a state defined. I think you'd need to concatenate lists of state undefined and state = present. Or you could ignore this micro optimisation for the state.